app/controllers/api/auth/stories_controller.rb

```diff
@@ -68,6 +81,19 @@ def search
   def search_params
     sparams = super
     sparams[:fq]['series_id'] = 'NULL' if filters.noseries?
+    sparams[:fq]['published_at'] = search_by_state(filters.state) if filters.state?
     sparams
   end
+
+  def search_by_state(state)
+    if state == 'published'
+      '[* TO now]'
```

Comment:
The inclusive/exclusive range syntax is documented [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#_ranges).